google/cloud/pubsub/samples/pubsub_samples_common.cc

```diff
@@ -188,6 +189,23 @@ std::string CommitSchemaRevisionsForRollbackSchemaTesting(
   return first_revision_id;
 }
 
+void CleanupSchemas(google::cloud::pubsub::SchemaServiceClient& schema_admin,
+                    std::string const& project_id, absl::Time const& time_now) {
+  auto const parent = google::cloud::Project(project_id).FullName();
+  for (auto& schema : schema_admin.ListSchemas(parent)) {
+    if (!schema) continue;
+    absl::Time schema_create_time =
+        absl::FromUnixSeconds((*schema).revision_create_time().seconds()) +
+        absl::Nanoseconds((*schema).revision_create_time().nanos());
```

Comment:
We have a helper:

https://github.com/googleapis/google-cloud-cpp/blob/654439aadc22fae0e8fcf8a030fca80d860c7182/google/cloud/internal/time_utils.h#L42

consider:

```suggestion
auto const schema_create_time =
    google::cloud::internal::ToAbslTime(schema->revision_create_time());
```